test/storage_snapper_configure_snapper_test.rb

```diff
@@ -22,13 +22,7 @@
 
     Yast::Storage.stub(:GetEntryForMountpoint).with("/").once.and_return(data)
 
-    Yast::SCR.stub(:Execute).and_return(1)
-    Yast::SCR.should_receive(:Execute).exactly(2).times.and_return(0)
-
-    Yast::SCR.stub(:Write).and_return(1)
-    Yast::SCR.should_receive(:Write).exactly(2).times.and_return(0)
-
-    Yast::StorageUtils.ConfigureSnapper()
+    expect(Yast::StorageSnapper.ConfigureSnapper?).to eq true
```

Comment:
To be honest, the original tests where not really that awesome :-), but this is clearly reducing the test coverage. Thus making things worse. ```ConfigureSnapper?``` is not the equivalent to the old ```ConfigureSnapper```. We should create tests for the real substitutes (Step1...Step4).

Since we are on it, the tests should be more accurate (not just checking that you call ```:Execute``` without actually checking what you are executing) and more descriptive (the current output looks like "ConfigureSnapper configures snapper. ConfigureSnapper does not configure snapper").